ecosystem/resources/members/quantum-prototype-template.toml

```diff
@@ -0,0 +1,15 @@
+name = "quantum-prototype-template"
+url = "https://github.com/qiskit-community/quantum-prototype-template"
+description = "A template repository for generating new quantum prototypes based on Qiskit."
+licence = "Apache License 2.0"
+contact_info = "_No response_"
+alternatives = "_No response_"
+affiliations = "Qiskit Community"
+labels = [ "plugin", "physics",]
```

Comment:
These labels don't exist any more; can you remove them and use the `Productivity` label instead please?

Reply:
Done, it's processing :)